### ISSUE TITLE
no-unused-vars missing require that isn't used

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#760]
I have this code at the beginning of a file:

``` js
var fs = require('fs'),
    path = require('path'),
    properties = require('properties'),
    assign = require('object-assign');
```

The `assign` variable is never used but isn't flagged (I found it through JSHint integration with my editor). Seems like we're missing the easiest case.
